build_library/vm_image_util.sh

```diff
@@ -594,9 +595,46 @@ label coreos
   kernel /coreos/vmlinuz
   append initrd=/coreos/cpio.gz coreos.autologin
 EOF
-    mkisofs -v -l -r -J -o $2 -b isolinux/isolinux.bin -c isolinux/boot.cat -no-emul-boot -boot-load-size 4 -boot-info-table .
-    isohybrid $2
+
+    # create fat image of grub for EFI
+    # 2880K floppy which fits our grub nicely.
+    dd if=/dev/zero of=isolinux/efi.img bs=512 count=5760 2>/dev/null
+    mformat -i isolinux/efi.img -v EFI -f 2880
```

Comment:
i don't think there was a requirement on size, it was just easier than figuring out the right C/H/S. :confounded: